roles/splunk_common/tasks/install_splunk.yml

```diff
@@ -22,7 +37,7 @@
 
 - name: Remove installer
   file:
-    dest: "{{ splunk.build_location }}"
+    dest: "/tmp/splunk.tgz"
```

Comment:
This destination still is required to be a flexible variable since Orca sends Splunk distribution into /opt/container_artifact directory.

Reply:
The problem with using "splunk.build_location" is that it can become a local build path or a custom build url. Otherwise, it will always be "/tmp/splunk.tgz"